src/dat/colours.json

```diff
@@ -0,0 +1 @@
 255}, "hsl": {"h": 0, "s": 0, "l": 100}, "name": "Grey100"}, "232": {"hexString": "#080808", "rgb": {"r": 8, "g": 8, "b": 8}, "hsl": {"h": 0, "s": 0, "l": 3}, "name": "Grey3"}, "233": {"hexString": "#121212", "rgb": {"r": 18, "g": 18, "b": 18}, "hsl": {"h": 0, "s": 0, "l": 7}, "name": "Grey7"}, "234": {"hexString": "#1c1c1c", "rgb": {"r": 28, "g": 28, "b": 28}, "hsl": {"h": 0, "s": 0, "l": 10}, "name": "Grey11"}, "235": {"hexString": "#262626", "rgb": {"r": 38, "g": 38, "b": 38}, "hsl": {"h": 0, "s": 0, "l": 14}, "name": "Grey15"}, "236": {"hexString": "#303030", "rgb": {"r": 48, "g": 48, "b": 48}, "hsl": {"h": 0, "s": 0, "l": 18}, "name": "Grey19"}, "237": {"hexString": "#3a3a3a", "rgb": {"r": 58, "g": 58, "b": 58}, "hsl": {"h": 0, "s": 0, "l": 22}, "name": "Grey23"}, "238": {"hexString": "#444444", "rgb": {"r": 68, "g": 68, "b": 68}, "hsl": {"h": 0, "s": 0, "l": 26}, "name": "Grey27"}, "239": {"hexString": "#4e4e4e", "rgb": {"r": 78, "g": 78, "b": 78}, "hsl": {"h": 0, "s": 0, "l": 30}, "name": "Grey30"}, "240": {"hexString": "#585858", "rgb": {"r": 88, "g": 88, "b": 88}, "hsl": {"h": 0, "s": 0, "l": 34}, "name": "Grey35"}, "241": {"hexString": "#626262", "rgb": {"r": 98, "g": 98, "b": 98}, "hsl": {"h": 0, "s": 0, "l": 37}, "name": "Grey39"}, "242": {"hexString": "#6c6c6c", "rgb": {"r": 108, "g": 108, "b": 108}, "hsl": {"h": 0, "s": 0, "l": 40}, "name": "Grey42"}, "243": {"hexString": "#767676", "rgb": {"r": 118, "g": 118, "b": 118}, "hsl": {"h": 0, "s": 0, "l": 46}, "name": "Grey46"}, "244": {"hexString": "#808080", "rgb": {"r": 128, "g": 128, "b": 128}, "hsl": {"h": 0, "s": 0, "l": 50}, "name": "Grey50"}, "245": {"hexString": "#8a8a8a", "rgb": {"r": 138, "g": 138, "b": 138}, "hsl": {"h": 0, "s": 0, "l": 54}, "name": "Grey54"}, "246": {"hexString": "#949494", "rgb": {"r": 148, "g": 148, "b": 148}, "hsl": {"h": 0, "s": 0, "l": 58}, "name": "Grey58"}, "247": {"hexString": "#9e9e9e", "rgb": {"r": 158, "g": 158, "b": 158}, "hsl": {"h": 0, "s": 0, "l": 61}, "name": "Grey62"}, "248": {"hexString": "#a8a8a8", "rgb": {"r": 168, "g": 168, "b": 168}, "hsl": {"h": 0, "s": 0, "l": 65}, "name": "Grey66"}, "249": {"hexString": "#b2b2b2", "rgb": {"r": 178, "g": 178, "b": 178}, "hsl": {"h": 0, "s": 0, "l": 69}, "name": "Grey70"}, "250": {"hexString": "#bcbcbc", "rgb": {"r": 188, "g": 188, "b": 188}, "hsl": {"h": 0, "s": 0, "l": 73}, "name": "Grey74"}, "251": {"hexString": "#c6c6c6", "rgb": {"r": 198, "g": 198, "b": 198}, "hsl": {"h": 0, "s": 0, "l": 77}, "name": "Grey78"}, "252": {"hexString": "#d0d0d0", "rgb": {"r": 208, "g": 208, "b": 208}, "hsl": {"h": 0, "s": 0, "l": 81}, "name": "Grey82"}, "253": {"hexString": "#dadada", "rgb": {"r": 218, "g": 218, "b": 218}, "hsl": {"h": 0, "s": 0, "l": 85}, "name": "Grey85"}, "254": {"hexString": "#e4e4e4", "rgb": {"r": 228, "g": 228, "b": 228}, "hsl": {"h": 0, "s": 0, "l": 89}, "name": "Grey89"}, "255": {"hexString": "#eeeeee", "rgb": {"r": 238, "g": 238, "b": 238}, "hsl": {"h": 0, "s": 0, "l": 93}, "name": "Grey93"}}
```

Comment:
format in human readable ????

Reply:
It's a constants file ripped from a reference, if a human edits this they're making a mistake, treat like a binary like everything else that'll go in /data